### PR TITLE
Us23 01 form contact

### PR DESCRIPTION
**Pull request by Wowlfy**

### CODE REVIEW

**Review by Loic-Code — DISMISSED** [review]

File: src/View/User/Includes/_contact_grey_block.html.twig [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+                                <label class="form-label">Email</label>
+                                <input type="email" class="form-control" >
+                            </div>
+                            <div class="mb-3">
+                                <label class="form-label">Message</label>
+                                <textarea class="form-control" rows="3"></textarea>
+                            </div>
+                        </div>
+                    </div>
+                {% endblock %}
```
Comment (LucyMerlier) [inline src/View/User/Includes/_contact_grey_block.html.twig]: indentation

File: src/View/User/Includes/_contact_grey_block.html.twig [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{% block contact_grey_block %}
+                    <div class="grey-block col-xl-3 offset-xl-8 col-sm-10 offset-sm-1">
+                        <div class="contact-grey-section">
+                        <div class="mb-3">
+                                <label class="form-label">Nom</label>
+                                <input type="nom" class="form-control" >
```
Comment (LucyMerlier) [inline src/View/User/Includes/_contact_grey_block.html.twig]: "nom" is not an input type

File: src/View/User/Includes/_contact_grey_block.html.twig [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+                            </div>
+                        <div class="mb-3">
+                                <label class="form-label">Prenom</label>
+                                <input type="prenom" class="form-control" >
```
Comment (LucyMerlier) [inline src/View/User/Includes/_contact_grey_block.html.twig]: "prenom" is not an input type

File: src/View/User/Includes/_contact_grey_block.html.twig [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+                            </div>
+                        <div class="mb-3">
+                                <label class="form-label">Telephone</label>
+                                <input type="Telephone" class="form-control" >
```
Comment (LucyMerlier) [inline src/View/User/Includes/_contact_grey_block.html.twig]: "Telephone" is not an input type

File: src/View/User/Includes/_contact_grey_block.html.twig [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+                            </div>
+                            <div class="mb-3">
+                                <label class="form-label">Email</label>
+                                <input type="email" class="form-control" >
+                            </div>
+                            <div class="mb-3">
+                                <label class="form-label">Message</label>
+                                <textarea class="form-control" rows="3"></textarea>
+                            </div>
```
Comment (LucyMerlier) [inline src/View/User/Includes/_contact_grey_block.html.twig]: no button to submit the form?

File: src/View/User/Includes/_contact_grey_block.html.twig [hunk 6]
```diff
@@ -0,0 +1,26 @@
+{% block contact_grey_block %}
+                    <div class="grey-block col-xl-3 offset-xl-8 col-sm-10 offset-sm-1">
+                        <div class="contact-grey-section">
+                        <div class="mb-3">
+                                <label class="form-label">Nom</label>
+                                <input type="nom" class="form-control" >
+                            </div>
```
Comment (LucyMerlier) [inline src/View/User/Includes/_contact_grey_block.html.twig]: no form tag?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: src/View/User/Includes/_contact_grey_block.html.twig (hunk 1 above)
Comment (Wowlfy) [reply src/View/User/Includes/_contact_grey_block.html.twig]: merci

File: src/View/User/Includes/_contact_grey_block.html.twig [hunk 7]
```diff
@@ -0,0 +1,29 @@
+{% block contact_grey_block %}
+    <form class="contact-grey-title" methode="POST" novalidate>
+        <div class="grey-block col-xl-3 offset-xl-8 col-sm-10 offset-sm-1">
+            <div class="contact-grey-section">
+                <div class="mb-3">
+                                    <label class="form-label">Nom</label>
```
Comment (bouboumael) [inline src/View/User/Includes/_contact_grey_block.html.twig]: indentation

File: src/View/User/Includes/_contact_grey_block.html.twig [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+                    <input type="text" class="form-control" name="nom">
+                </div>
+                <div class="mb-3">
+                    <label class="form-label">Prenom</label>
+                    <input type="text" class="form-control" name="prenom">
+                </div>
+                <div class="mb-3">
+                    <label class="form-label">Telephone</label>
```
Comment (bouboumael) [inline src/View/User/Includes/_contact_grey_block.html.twig]: your labels are not linked to your inputs. use for = "xxx" in labels and id = "xxx" in entries

File: src/View/User/Includes/_contact_grey_block.html.twig [hunk 9]
```diff
@@ -0,0 +1,29 @@
+{% block contact_grey_block %}
+    <form class="contact-grey-title" methode="POST" novalidate>
+        <div class="grey-block col-xl-3 offset-xl-8 col-sm-10 offset-sm-1">
+            <div class="contact-grey-section">
+                <div class="mb-3">
+                                    <label class="form-label">Nom</label>
+                    <input type="text" class="form-control" name="nom">
+                </div>
+                <div class="mb-3">
+                    <label class="form-label">Prenom</label>
```
Comment (bouboumael) [inline src/View/User/Includes/_contact_grey_block.html.twig]: your labels are not linked to your inputs. use for = "xxx" in labels and id = "xxx" in entries

File: src/View/User/Includes/_contact_grey_block.html.twig [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+                    <input type="text" class="form-control" name="prenom">
+                </div>
+                <div class="mb-3">
+                    <label class="form-label">Telephone</label>
+                    <input type="text" class="form-control" name="telephone">
+                </div>
+                <div class="mb-3">
+                    <label class="form-label">Email</label>
```
Comment (bouboumael) [inline src/View/User/Includes/_contact_grey_block.html.twig]: the same as above

File: src/View/User/Includes/_contact_grey_block.html.twig [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+                    <input type="email" class="form-control" name="email">
+                </div>
+                <div class="mb-3">
+                        <label class="form-label">Message</label>
```
Comment (bouboumael) [inline src/View/User/Includes/_contact_grey_block.html.twig]: the same as above

**Review by bouboumael — CHANGES_REQUESTED** [review]

**Review by bouboumael — CHANGES_REQUESTED** [review]
Please delete and adding .DS_Store into .gitignore.
With this file your have 8 files changed...

**Review by bouboumael — DISMISSED** [review]
6 files because fix .ds_store :)

File: src/View/User/Includes/_contact_grey_block.html.twig [hunk 12]
```diff
@@ -0,0 +1,29 @@
+{% block contact_grey_block %}
+    <form class="contact-grey-title" methode="POST" novalidate>
```
Comment (LucyMerlier) [inline src/View/User/Includes/_contact_grey_block.html.twig]: novalidate are only for testing, don't forget to take them off before pushing

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comment

**Review by bouboumael — DISMISSED** [review]

**Review by LucyMerlier — DISMISSED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**